core/src/main/java/tc/oc/pgm/filters/dynamic/FilterMatchModule.java

```diff
@@ -89,10 +93,22 @@ public FilterMatchModule(Match match) {
   // Filterables that need a check in the next tick (cleared every tick)
   private final Set<Filterable<?>> dirtySet = new HashSet<>();
 
-  // We have to listen to this event to guarantee that all other modules have been loaded
-  // and that there is no unresolved xml filter references.
-  @EventHandler
-  public void onMatchLoad(MatchLoadEvent event) {
+  // See MatchImpl#load
+  public void onMatchLoad() {
+
+    // First, find and create all necessary reactors
+    this.context
+        .getAll(FilterDefinition.class)
+        .forEach(
+            filter -> {
+              for (Filter filt3r : Filters.flattenFilter(filter, new ArrayList<>())) {
```

Comment:
Im sorry what. What is filt3r. just name it f, or child

```suggestion
              for (Filter child : Filters.flattenFilter(filter, new ArrayList<>())) {
```